### DISCUSSION TITLE
“Reflector 2” will damage your computer. You should move it to the Trash.

### DISCUSSION THREAD

**Discussion by AnastasiusVivaldus** [Homebrew/discussions#2229] [Casks]
Just ran `brew install reflector2` and got this message from my Mac. I assume that the app is safe but that there was an error somewhere in the signature-verification pipeline. Not sure how to proceed. Shall I report this to homebrew via a new github issue?

**Comment by SMillerDev** [comment]
there is no such formula provided by the homebrew project, please create an issue for with the tap that hosts this software.

**Reply by AnastasiusVivaldus** [reply]
@SMillerDev OK, for my own edification, I thought that `brew install reflector2` is a formula. How can I distinguish between 'formula' and 'tap'?

**Reply by SMillerDev** [reply]
A tap is a repository hosting software, which can be Formula or Casks. https://github.com/Homebrew/homebrew-core and https://github.com/Homebrew/homebrew-cask are the default taps. But anyone can host their own.

https://docs.brew.sh/Formula-Cookbook#homebrew-terminology has a list of the naming.

**Reply by AnastasiusVivaldus** [reply]
Sorry, still not following. I don't really understand in any detail how the homebrew ecosystem works, and this list self-referential terms doesn't make sense to me.

In this case, when I run `brew info reflector2` it tells me:

```
❯ brew info reflector2
reflector2: 2.7.3
https://www.airsquirrels.com/reflector/
/usr/local/Caskroom/reflector2/2.7.3 (125B)
From: https://github.com/Homebrew/homebrew-cask-versions/blob/HEAD/Casks/reflector2.rb
==> Name
Reflector
==> Description
Wireless screen-mirroring application
==> Artifacts
Reflector 2.app (App)
```

... so when you say "please create an issue for with the tap that hosts this software", do you mean to open an issue at the repo `https://github.com/Homebrew/homebrew-cask-versions` (in order to adjust the reflector2.rb file), or do you mean to contact the host of `www.airsquirrels.com` whence this software is actually downloaded?
